bin/xud

```diff
@@ -57,8 +57,13 @@ const { argv } = require('yargs')
       type: 'boolean',
       default: undefined,
     },
-    nosanitychecks: {
-      describe: 'Whether to disable order sanity checks',
+    sanityswaps: {
+      describe: 'Whether to enable sanity swaps',
+      type: 'boolean',
+      default: undefined,
+    },
+    balancechecks: {
```

Comment:
I think it might be good to keep the "no" in the config option naming if the goal long term is to enable these by default, which is my understanding. Otherwise we may want to rename them in the future and that would be a hassle if people need to rewrite their confg files. So I'd go with `nosanityswaps` and `nobalancechecks`.

Reply:
The reason I opted to go for `balancechecks` and `sanityswaps` was that for me it felt very weird to read double negative code. Consider the previous code snippet:
```
if (!this.nosanitychecks) {
```
This reads in my head as if no-no-sanitychecks which causes confusion whereas `if (this.sanitychecks)` or `if (!this.sanitychecks)` is much clear to me.

Reply:
That's a fair point, but I'm thinking mainly about the implications for passing args to xud. I think it's more convenient to be able to do `xud --nosanitychecks` as opposed to `xud --sanitychecks false`. Internally at some point we could `sanitychecks = !nosanitychecks` and use that variable in our logic.

Reply:
I went ahead and changed it to `nosanityswaps` and `nobalancechecks`. I feel like adding an abstraction layer between the CLI flags and the actual code would add unnecessary complexity.